test/ServiceTest.cpp

```diff
@@ -155,6 +161,8 @@ TEST_F(ServiceTest, computeResult) {
       getTsvFunctionFactory(
           expectedUrl, expectedSparqlQuery,
           "?x\t?y\n<x>\t<y>\n<bla>\t<bli>\n<blu>\t<bla>\n<bli>\t<blu>\n")};
+  serviceOperation4.createRuntimeInfoFromEstimates(
+      std::make_shared<RuntimeInformation>());
```

Comment:
Also: why6 do you sometimes use this pattern, and sometimes only `isRoot() = true`?